plugins/inputs/systemd_units/systemd_units_linux.go

```diff
@@ -24,6 +24,132 @@ type systemctl func(Timeout internal.Duration, UnitType string) (*bytes.Buffer,
 
 const measurement = "systemd_units"
 
+// unit_load_state_table, see
+// https://github.com/systemd/systemd/blob/c87700a1335f489be31cd3549927da68b5638819/src/basic/unit-def.c#L87
+var load_map = map[string]int{
+	// err: 0,
```

Comment:
Let's remove all the commented out values, if they aren't being used, it doesn't seem valuable to have lingering commented code lying around.